chat-plugins/modlog.js

```diff
@@ -333,14 +394,24 @@ exports.commands = {
 			roomIdList = [roomId];
 		}
 
-		PM.send(roomIdList.join(','), searchString, exactSearch, lines).then(response => {
-			connection.send(prettifyResults(response, roomId, searchString, exactSearch === '1', addModlogLinks, hideIps, lines));
+		PM.send(roomIdList.join(','), searchString, lines).then(response => {
+			connection.send(prettifyResults(response, roomId, searchString, response.charAt(0), addModlogLinks, hideIps, lines));
 			if (cmd === 'timedmodlog') this.sendReply(`The modlog query took ${Date.now() - startTime} ms to complete.`);
 		});
 	},
-	modloghelp: [
-		"/modlog [roomid], [search] - Searches the moderator log - defaults to the current room unless specified otherwise.",
-		"If you set [roomid] as [all], it searches for [search] on all rooms' moderator logs.",
-		"If you set [roomid] as [public], it searches for [search] in all public rooms' moderator logs, excluding battles. Requires: % @ * # & ~",
-	],
+	modloghelp: function (target, room, user) {
+		if (!this.runBroadcast()) return false;
+
+		this.sendReplyBox(
+			`/modlog [roomid], [search] - Searches the moderator log - defaults to the current room unless specified otherwise.<br />` +
+			`- If you set [roomid] as [all], it searches for [search] on all rooms' moderator logs.<br />` +
+			`- If you set [roomid] as [public], it searches for [search] in all public rooms' moderator logs, excluding battles. Requires: % @ * # & ~<br />` +
+			`- Put <code style='border: 1px lightblue solid; background: rgba(0, 50, 0, 0.02); padding-left: 3px; padding-right: 3px'>"</code> around any search term to get the exact string, with symbols and spacing included, in any part of the modlog entry.  By default, it will search for complete words or phrases only if quotation marks are not used.<br />` +
+			`- Use <code style='border: 1px lightblue solid; background: rgba(0, 50, 0, 0.02); padding-left: 3px; padding-right: 3px'>&&</code> include additional search terms.  Results will include lines with all the terms.<br />` +
+			`- Use <code style='border: 1px lightblue solid; background: rgba(0, 50, 0, 0.02); padding-left: 3px; padding-right: 3px'>||</code> to include additional search terms. Results will include lines with any or all of the terms.<br />` +
+			`- Use <code style='border: 1px lightblue solid; background: rgba(0, 50, 0, 0.02); padding-left: 3px; padding-right: 3px'>!</code> to exclude any search term.  Results will not include lines containing that term. <br />` +
```

Comment:
No need for a space after the period and before the ``<br />`` towards the end of this line

Reply:
Also, if you're going to insist on using inline CSS here, instead of repeating it over and over, can you pls declare it as a variable since this is a command

Reply:
finally thanks!
i've been searching for the stuff to make it appear as if it was in chat.